lib/hooks/create/project.js

```diff
@@ -27,26 +39,35 @@ module.exports = settings => {
             if (project.status === 'inactive') {
               return model.setStatus(autoResolved.id);
             } else {
-              return Promise.resolve()
-                .then(() => canEndorse(model))
-                .then(can => model.setStatus(can ? endorsed.id : awaitingEndorsement.id));
+              return model.setStatus(withInspectorate.id);
             }
           });
 
       case 'grant':
-        return messager({ ...model.data, action: 'submit-draft' })
-          .then(() => canEndorse(model))
-          .then(can => model.setStatus(can ? endorsed.id : awaitingEndorsement.id));
+        return Promise.resolve()
+          .then(needsEndorsing => {
```

Comment:
isn't this always `undefined`?

Reply:
also it doesn't look like it is used so will prob fail lint

Reply:
It looks a lot like I did a half-arsed copy-paste there doesn't it?